docs/live-migration.md

```diff
@@ -0,0 +1,29 @@
+# Live Migration
+
+Live-migrating a virtual machine means moving it to a different host while the guest workload continues to run and remain accessible.
```

Comment:
prefer to simply with: 
Live migration means that users can now migrate one VM from one host to another host without downtime.